config.go

```diff
@@ -0,0 +1,44 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"io/ioutil"
+
+	yaml "gopkg.in/yaml.v2"
+)
+
+// DefaultConfigPath refers to location of configuration mount dir specified in deployment
+const DefaultConfigPath = "/etc/webhook/config/config.yaml"
+
+// Config represents all config we need to initialize the webhook server
+type Config struct {
+	Trace struct {
```

Comment:
I think this means the `Trace` struct is embedded in the `Config` struct, meaning you can do config.SampleRate.  Can we instead explicitly name the trace struct subfield, so it is clear what the yaml key is for it?  I.E.
```
type Config struct {
    Trace Trace `yaml:"trace"`
}

// Trace is the configuration for the trace context added to pods
type Trace struct {
    SampleRate float64 `yaml:"sampleRate"`
}
```

Reply:
done